app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -218,6 +221,16 @@ class UrlInputFragment :
         }
     }
 
+    private fun animateWordMark() {
+        keyboardLinearLayout.alpha = 0f
+        keyboardLinearLayout.animate()
+                .setInterpolator(AccelerateDecelerateInterpolator())
+                .alpha(0f)
+                .alphaBy(1f)
+                .setDuration(WORD_MARK_ANIMATION_DURATION.toLong())
+                .start()
+    }
+
```

Comment:
Could this be added to the animation code we already have in place for the translation animation of the word mark?